src/Styleguide/Components/__stories__/ReadMore.story.tsx

```diff
@@ -10,7 +10,7 @@ storiesOf("Styleguide/Components", module).add("ReadMore", () => {
     <React.Fragment>
       <Section title="Max lines">
         <Item>
-          <Sans>
+          <Sans size="3">
```

Comment:
Wondering if there is still a value in allowing one not to specify the size and in this case just inherit the value from a parent. That might come handy when nesting one Sans within the other to just update the font color or something along those lines. 

That would allow us to save a bit of definition in cases like that for example: https://github.com/artsy/reaction/blob/master/src/Styleguide/Pages/Artwork/Sidebar/TitleInfo.tsx#L24

Not that it's really hard to repeat couple of sizes:)